src/desktop/DesktopMain.js

```diff
@@ -82,6 +82,14 @@ if (process.argv.indexOf("-r") !== -1) {
 	            .then(() => app.exit(0))
 	            .catch(() => app.exit(1))
 } else {
+	startupInstance()
+}
+
+function startupInstance() {
+	// Delete the temp directory on startup, because we may not always be able to do it on shutdown
+	// we want to await cause the next call will create it again
```

Comment:
"we want to await" but no await is confusing

Reply:
oops old comment made obsolete with changes, will delete